docroot/themes/humsci/humsci_basic/src/scss/utilities/_lists.scss

```diff
@@ -0,0 +1,21 @@
+.hb-pill-list {
+  ol,
+  ul {
+    padding: 0;
+    margin: 0;
+
+    li {
+      display: inline-block;
+      margin-right: hb-calculate-rems(8px);
+
+      &::before {
+        display: none;
+      }
+    }
+  }
+
+  a {
```

Comment:
I'm not sure if this rule is necessary. They should already have a pill class on the anchor tag if they want it to be styled as such.